src/Security/test/Authentication.CloudFoundryBase.Test/CloudFoundryTokenValidatorTest.cs

```diff
@@ -19,4 +20,44 @@ public void ValidateIssuer_ValidatesCorrectly()
         Assert.NotNull(uaaResult);
         Assert.Null(foobarResult);
     }
+
+    [Fact]
+    public void ValidateAudience_ValidatesFromAuthServerOptionsCorrectly()
+    {
+        var cftv = new CloudFoundryTokenValidator(new AuthServerOptions
+        {
+            ClientId = "test-client",
+            AdditionalAudiences = new[] { "additional-audience" }
+        });
+        var audiences = new[] { "profile", "some-api", "additional-audience" };
+        var validAudience = cftv.ValidateAudience(audiences, null, null);
+        Assert.True(validAudience);
+
+        audiences = new[] { "invalid-audience" };
+        var invalidAudience = cftv.ValidateAudience(audiences, null, null);
```

Comment:
It looks like this (and above) encodes the expected outcome in the variable name. Please don't do that; instead, use a variable name such as `isValid`, `result`, `validateResult`, etc.